### DISCUSSION TITLE
"Credit card" type asset account not appearing in API endpoint account list.

### DISCUSSION THREAD

**Discussion by digitlength** [firefly-iii/firefly-iii#8847] [Firefly III] [answered]
When querying

```
/v1/chart/account/overview
```

with appropriate date parameters set, all asset account data is returned except for the one where the "Account role" is set to "Credit card".

Is this by design?

**Comment by JC5** [comment, marked as answer]
This particular endpoint gets it list from the "Home screen" preference on your `/preferences` page. If it's present in that particular list, it shows up. I just tested this, and it works as expected.

**Reply by digitlength** [reply]
I've changed my preferences now - that account was indeed unchecked.

Might be worth adding this detail into the description in the API documentation: https://api-docs.firefly-iii.org/#/charts/getChartAccountOverview
